cpp/src/stream_compaction/unique_count_column.cu

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2022-2023, NVIDIA CORPORATION.
```

Comment:
should be copyright be 2023 only? how is that supposed do work for copied code?

Reply:
Since the code is moved I thought it should have the date for the source text itself. But the copyright may actually go with the file itself. I will change it to just 2023.